src/types.ts

```diff
@@ -2122,6 +2122,19 @@ export type User<UserType = UnknownType> = UserType & {
   username?: string;
 };
 
+export type DeleteType = 'soft' | 'hard';
+
+export type DeleteUserOptions = {
+  user_ids: string[];
+  conversations?: DeleteType;
+  messages?: DeleteType;
+  user?: DeleteType;
```

Comment:
you mean it should be required?
```suggestion
  user_ids: string[];
  user: DeleteType;
  conversations?: DeleteType;
  messages?: DeleteType;
```

Reply:
isn't that what I did?

Reply:
it's a bit confusing how the behavior is on this one, if no option is passed is it soft delete or no delete for `messages` and `conversations`? also what's the difference between these two?

Reply:
> isn't that what I did?

it was still showing it as optional!

Reply:
Seems confusing to me to make "deleting a user" optional, on a Deleteusers endpoint. However, messages and conversations are optional.

Reply:
what exactly is `conversations`?

Reply:
1on1 channels

Reply:
any with 2 members, so after deleting the user we'd be left with a channel with only 1 member, in which case we delete it (with this option)

Reply:
makes sense, should we add some docs to clarify this in the type?